packages/obojobo-document-engine/server/attempt-end.js

```diff
@@ -64,15 +63,15 @@ const endAttempt = (req, res, user, draftDocument, attemptId, isPreview) => {
 
 			assessmentScoreId = completeAttemptResult.assessmentScoreId
 
-			return reloadAttemptStateIfReviewing(
-				attemptId,
-				attempt.draftId,
-				attempt,
-				draftDocument,
-				user,
-				isPreview,
-				attemptHistory
-			)
+			// return reloadAttemptStateIfReviewing(
```

Comment:
This is no longer used, but I want to keep it in the PR in case I unknowingly deleted important functionality. Will remove be pull.